overlord/snapstate/booted.go

```diff
@@ -130,7 +130,7 @@ func CurrentBootNameAndRevision(typ snap.Type) (name string, revision snap.Revis
 	case snap.TypeKernel:
 		kind = "kernel"
 		bootVar = "snap_kernel"
-	case snap.TypeOS:
+	case snap.TypeOS, snap.TypeBase:
 		kind = "core"
```

Comment:
this goes into the error messages,  for base we should set it to "bootable base" I think